frontend/src/i18n/locales/zh-CN.json

```
@@ -43,7 +43,9 @@
     "off": "关",
     "about": "简介",
     "built-by": "{0} 项目",
-    "links": "更多"
+    "links": "更多",
+    "send-history": "发送历史记录",
```

```suggestion
    "send-history": "发送历史",
```
More concise with same meaning

---

https://github.com/ScopeLift/umbra-protocol/pull/560/commits/7582b901fc9dd942e140309eb7f80a9d59edfbde